frontend/packages/dev-console/integration-tests/features/operators.feature

```diff
@@ -93,6 +93,16 @@ Scenario: Create HyperConverged Cluster: VM-01-TC02
     Then user will see a HyperConverged Cluster created
     And user will see Virtualization item under Workloads
 
+@regression @smoke
+Scenario: Quay container security operator
+   Given user is at Operator Hub page with the header name "OperatorHub"
+   When user searches for "quay container security"
+   And user clicks on quay container security operator card on Operator Hub page
+   And user clicks install button present on the right sidebar
```

Comment:
```suggestion
   And user clicks on install button on sidebar
```

Reply:
I've used it for uniformity with other operator installation mentioned in the doc